app/main.js

```diff
@@ -176,21 +177,22 @@ const Timer = React.createClass({
     }
   },
   pause: function() {
+    currentTask.paused = true
     clearInterval(this.intervalSec)
     clearInterval(this.intervalMin)
   },
   togl: function() {
-    if(!this.start) {
-        this.start
+    if(currentTask.paused === 'true') {
```

Comment:
Same thing with the above comment [1] here. You need to find the `currentTask` first.

Also, some thing with true. Just `true`, not `'true'` (no quotes)

[1] https://github.com/sirodoht/ketchup/pull/5/files/4643cd197cc477d785a032da8b8c0d3c96101715..6441c0dbe1e239278ea4194a25d320719f3f3978#r105179502